webapp/packages/plugin-sql-editor-navigation-tab/src/SqlEditorTab.tsx

```diff
@@ -59,8 +61,30 @@ export const SqlEditorTab: TabHandlerTabComponent<ISqlEditorTabState> = observer
   const handleSelect = ({ tabId }: ITabData<any>) => onSelect(tabId);
   const handleClose = onClose ? ({ tabId }: ITabData<any>) => onClose(tabId) : undefined;
 
+  let tooltip = name;
+
+  if (connection) {
+    tooltip += `\n${translate('ui_connection')}: ${connection.name}`;
+  }
+
+  if (executionContext) {
+    const project = projectInfo.get(executionContext.projectId);
+
+    if (executionContext.defaultCatalog) {
+      tooltip += `\n${translate('ui_catalog')}: ${executionContext.defaultCatalog}`;
+    }
+
+    if (executionContext.defaultSchema) {
+      tooltip += `\n${translate('ui_schema')}: ${executionContext.defaultSchema}`;
+    }
+
+    if (project) {
+      tooltip += `\n${translate('ui_project')}: ${project.name}`;
+    }
+  }
```

Comment:
this part of logic is duplicated, can we move it to separate function-helper that will return tooltip string (localized)
something like `getObjectInfoTooltip(translate, connection?, catalog?, schema?, project?)`